cmd/ndm_daemonset/probe/eventhandler.go

```diff
@@ -149,3 +173,7 @@ func isParentOrSlaveDevice(bd blockdevice.BlockDevice, erroredDevices []string)
 	}
 	return false
 }
+
+func isAllBlockDevices(devices []*blockdevice.BlockDevice) bool {
+	return len(devices) == 1 && devices[0] == &blockdevice.AllBlockDevices
```

Comment:
Didnt understand the logic being implemented here.

Reply:
so we have a variable in blockdevice.go named `AllBlockDevices`. When we want to denote that check all existing block devices, we just pass the address of that variable in the device slice. The same is being checked here.

Reply:
Also, instead of comparing addresses can some other approach be used.

Reply:
We can also check for a particular field in the block device, say the dev path. If the dev path is set to a particular value, then we can say that it means All block devices.

Reply:
should I change to this approach?

Reply:
Can we have some other field, rather than depending on the fields in blockdevice struct.